pvr.sledovanitv.cz/addon.xml.in

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.sledovanitv.cz"
-  version="2.4.1"
-  name="PVR sledovanitv.cz(unofficial)"
+  version="1.0.0"
```

Comment:
Should I remove/rename the existing tags from git? (does the kodi source/team even care about release tags in external addons?)

Reply:
For stable kodi branches we take care of release tags, yes. For addon versions for kodi master branch, no tags are needed

Reply:
Examples:

https://github.com/xbmc/repo-binary-addons/blob/master/pvr.octonet/pvr.octonet.txt

vs.

https://github.com/xbmc/repo-binary-addons/blob/Krypton/pvr.octonet/pvr.octonet.txt

Reply:
Done. And created the 1.0.0 tag...